.gitignore

```diff
@@ -5,3 +5,4 @@ vendor
 *.log
 tags
 .DS_Store
+.tern-project
```

Comment:
[Tern](http://ternjs.net/) is a code analysis tool.  It integrates with sublime, emacs, vim ...etc and gives you auto complete and 'goToDef' functionality.  You need a .tern-project file to add plugin support etc.